cirq-core/cirq/contrib/requirements.txt

```diff
@@ -7,6 +7,8 @@ pylatex~=1.3.0
 quimb
 opt_einsum
 autoray
+# required for py39 opcodes.
+numba>=0.53.0
```

Comment:
I just realized that we should have conditioned this on python3.9...We want to add the least amount of restrictions in general.